Few man pages modifications for Solaris.

Note that xsltproc.xml used for generation of xsltproc.1 needs to be
tweaked little bit to fit output for Solaris nroff.

--- libxslt-1.1.27/libxslt/libxslt.3	Mon May 11 23:29:34 2009
+++ libxslt-1.1.27/libxslt/libxslt.3	Tue Oct  9 04:34:52 2012
@@ -10,13 +10,10 @@
 .LP
 .SH FILES
 .TP 2.2i
-.B /usr/lib/libxslt_1.0.0/libxslt.a
-static library
-.TP
-.B /usr/lib/libxslt_1.0.0/libxslt.so
+.B /usr/lib/libxslt.so
 sharable library
 .TP
-.B /usr/package/libxslt_1.0.0/bin/xsltproc
+.B /usr/bin/xsltproc
 binary application to do XSL transformations on the command line
 .SH AUTHORS
 Daniel Veillard (daniel@veillard.com).
--- libxslt-1.1.27/libexslt/libexslt.3	Mon May 11 23:29:34 2009
+++ libxslt-1.1.27/libexslt/libexslt.3	Tue Oct  9 04:44:18 2012
@@ -255,9 +255,6 @@
 .I /usr/bin/xslt-config
 shell script giving pre-processor and linker flags.
 .TP
-.I /usr/lib/libexslt.a
-static library
-.TP
 .I /usr/lib/libexslt.so
 sharable library
 .SH AUTHORS
--- libxslt-1.1.27/doc/xsltproc.xml	Tue Sep 11 23:35:50 2012
+++ libxslt-1.1.27/doc/xsltproc.xml	Thu Oct 11 01:45:40 2012
@@ -26,7 +26,7 @@
 		</affiliation>
 	</author>
 	<!-- date should be the date of the latest change or the release version -->
-	<date>$Date$</date>
+	<date>12 Sep 2012</date>
 	<!-- still a bit buggy output, will talk to docbook-xsl upstream to fix this -->
 	<!-- <releaseinfo>This is release 0.4 of the xsltproc Manual.</releaseinfo> -->
 	<!-- <edition>0.4</edition> -->
@@ -408,7 +408,7 @@
 		<para>
 			Process the input document using the XInclude specification. More
 			details on this can be found in the XInclude
-			specification: <ulink url="http://www.w3.org/TR/xinclude/"/>
+			specification: http://www.w3.org/TR/xinclude/
 		</para>
 	</listitem>
 		</varlistentry>
@@ -559,11 +559,11 @@
 				<para><citerefentry>
 						<refentrytitle>libxml</refentrytitle>
 						<manvolnum>3</manvolnum>
-					</citerefentry> web page <ulink url="http://www.xmlsoft.org/"/>
+					</citerefentry> web page http://www.xmlsoft.org/
 				</para>
 			</listitem>
 			<listitem>
-				<para>W3C <acronym>XSLT</acronym> page <ulink url="http://www.w3.org/TR/xslt"/>
+				<para>W3C <acronym>XSLT</acronym> http://www.w3.org/TR/xslt
 				</para>
 			</listitem>
 		</itemizedlist>
